packages/dotnet/Cisco.Opentelemetry.Specifications/trace/PayloadAttributes.cs

```diff
@@ -30,9 +30,13 @@ public static class PayloadAttributes {
       "aws.sqs.result_entry",
       "aws.sqs.record.attribute",
       "aws.sqs.record.message_attribute",
+      "rpc.request.metadata",
```

Comment:
it seems a bit strange that "request.metadata" is considered as data (and not as metadata).
Is that intended?

Reply:
grps.metadata is the equivalent of http.headers - it lets the client provide information associated with the call to the server and vice versa.